solidity/test/unit/Greeter.t.sol

```diff
@@ -47,7 +47,9 @@ contract Unit_Greeter_SetGreeting is Base {
         vm.startPrank(owner);
     }
 
-    function test_RevertIfNotOwner(address caller, string memory greeting) public {
+    function test_RevertIfNotOwner(address caller, string memory greeting)
+        public
```

Comment:
Are we okay with this format? The public being alone seems slightly weird to me, but it could be lack of custom